apps/next/util/database/paymentHelpers.ts

```diff
@@ -0,0 +1,75 @@
+import prisma from '../ssr/prisma';
+
+// add any other payment logic to this file in the future
+export type PaymentPlan = 'Professional' | 'Personal' | 'Free';
+
+export type APIPaymentObject = {
+    plan: PaymentPlan;
+    partner: boolean;
+};
+
+const checkPartnerAccount = async (userId: string): Promise<boolean> => {
+    const userInfo = await prisma.user?.findFirst({
+        where: {
+            id: userId,
+        },
+        select: {
+            partner: true,
+        },
+    });
+
+    return userInfo.partner ?? false;
+};
+
+const checkPaymentPlan = async (userId: string): Promise<PaymentPlan> => {
```

Comment:
It's a little hard to tell how this function `checkPaymentPlan` is different than `productPlan` function below. They have different names, but I think we need a comment at least explaining each one.